diff -Nru a/src/3rdparty/3rdparty.pro b/src/3rdparty/3rdparty.pro
--- a/src/3rdparty/3rdparty.pro	2018-01-04 12:06:28.000000000 +0100
+++ b/src/3rdparty/3rdparty.pro	2018-01-13 15:04:57.672244620 +0100
@@ -2,8 +2,7 @@
 TEMPLATE = subdirs
 SUBDIRS = \
     spatialindexnavel \
-    stemmer \
-    quazip
+    stemmer
 
 !r_no_opennurbs {
     SUBDIRS += opennurbs
diff -Nru a/src/zip/zip.pro b/src/zip/zip.pro
--- a/src/zip/zip.pro	2018-01-04 12:06:28.000000000 +0100
+++ b/src/zip/zip.pro	2018-01-13 15:07:42.170233519 +0100
@@ -1,6 +1,6 @@
 include( ../../shared.pri )
 
-INCLUDEPATH+=../3rdparty/quazip/src
+INCLUDEPATH+=/usr/include/quazip
 
 greaterThan(QT_MAJOR_VERSION, 4) {
     INCLUDEPATH += $$[QT_INSTALL_HEADERS]/QtZlib
