--- qcad-2.0.5.0-1-community.src/qcad/src/qc_applicationwindow.cpp.path	2006-05-05 16:06:07.652672917 +0200
+++ qcad-2.0.5.0-1-community.src/qcad/src/qc_applicationwindow.cpp	2006-05-05 16:07:31.800460482 +0200
@@ -2946,12 +2946,12 @@
                         RS_SYSTEM->getAppDir().latin1());
         RS_DEBUG->print("QC_ApplicationWindow::slotHelpManual(): appdir: %s",
                         RS_SYSTEM->getAppDir().latin1());
-        assistant = new QAssistantClient(RS_SYSTEM->getAppDir()+"/bin", this);
+        assistant = new QAssistantClient("@BINDIR@", this);
 		connect(assistant, SIGNAL(error(const QString&)), 
 			this, SLOT(slotError(const QString&)));
         QStringList args;
         args << "-profile";
-        args << QDir::convertSeparators(RS_SYSTEM->getDocPath() + "/qcaddoc.adp");
+        args << QDir::convertSeparators("@DATADIR@/qcad/doc/qcaddoc.adp");
 //        args << QString("doc") + QDir::separator() + QString("qcaddoc.adp");
 
 #if QT_VERSION>=0x030200 
