--- qcad-2.0.4.0-1.src/qcad-manual-en-2.0.4.0-1.html/qcaddoc.adp.manfix	2004-09-12 00:51:05.000000000 +0200
+++ qcad-2.0.4.0-1.src/qcad-manual-en-2.0.4.0-1.html/qcaddoc.adp	2006-05-05 14:39:17.148021289 +0200
@@ -4,8 +4,6 @@
 
     <property name="title">QCad Manual</property>
 
-    <property name="applicationicon">cad/imgs/qcadicon.png</property>
-
     <property name="startpage">cad/index.html</property>
 
     <property name="aboutmenutext">About QCad Manual</property>
@@ -15,7 +13,7 @@
     <property name="assistantdocs">.</property>
   </profile>
 
-  <DCF ref="cad/index.html" icon="cad/imgs/manual.png" title="QCad Manual">
+  <DCF ref="cad/index.html" title="QCad Manual">
     <section ref="cad/index.html" title="Home">
     </section>
 
